src/base/Query.sol

```diff
@@ -36,13 +37,33 @@ abstract contract Query is IQuery {
     {
         AttestationRecord storage attestation = _getAttestation(module, attester);
 
-        uint256 expirationTime = attestation.expirationTime;
-        attestedAt = expirationTime != ZERO_TIMESTAMP && expirationTime < block.timestamp
-            ? ZERO_TIMESTAMP
-            : attestation.time;
-        if (attestedAt == ZERO_TIMESTAMP) revert AttestationNotFound();
+        // attestedAt = attestation.time;
+        uint256 expirationTime; // = attestation.expirationTime;
+        uint256 revocationTime; // = attestation.revocationTime;
+
+        // @dev this loads the three times from storage, bit shifts them and assigns them to the variables
```

Comment:
"three times" reads a bit weird. should maybe say

_this loads the 3 time variables from storage _